### ISSUE TITLE
Change gradSchemes from leastSquares to Gauss linear for 2D gases

### ISSUE THREAD

**Issue by TheodoreGalanos** [ladybug-tools/butterfly#293]
LS will give an error on 2D meshes, I believe due to the way it is calculated.

**Comment by TheodoreGalanos** [comment]
@mostaphaRoudsari Is this implemented already, should I close?

**Comment by mostaphaRoudsari** [comment]
@TheodoreGalanos, no! it's not. This is an easy one to implement if you want to start getting involved with modifying the code.

**Comment by MStreet3** [comment]
@mostaphaRoudsari it seems this may still be open.  If it is, where in the code base is the solver for 2D gases implemented?  I'd like to take a look.
